mods/ra/weapons/smallcaliber.yaml

```diff
@@ -201,16 +201,24 @@ ChainGun:
 		Versus:
 			None: 120
 
-ChainGun.Yak:
+ChainGun.Yak.Right:
 	Inherits: ^HeavyMG
-	ReloadDelay: 3
-	Range: 5c0
-	MinRange: 3c0
+	Burst: 5
```

Comment:
This won't evenly divide into 18, but nothing will so we can leave this for a followup balance patch to hopefully iron out.